docs/guide/export.rst

```diff
@@ -0,0 +1,65 @@
+.. _export:
+
+
+Exporting models
+================
+
+After training an agent, you may want to deploy/use it in an other language
+or framework, like `tensorflowjs <https://github.com/tensorflow/tfjs>`_.
+Stable Baselines3 does not include tools to export models to other frameworks, but
+this document aims to cover parts that are required for exporting along with
+more detailed stories from users of Stable Baselines3.
+
+
+Background
+----------
+
+In Stable Baselines3, the controller is stored inside policies which convert
+observations into actions. Each learning algorithm (e.g. DQN, A2C, SAC) contains
+one policy, accessible via ``model.policy``.
+
+Policies hold enough information to do the inference (i.e. predict actions),
+so it is enough to export these policies (cf :ref:`examples <examples>`)
+to do inference in an another framework.
+
+.. warning::
+  When using CNN policies, the observation is normalized during pre-preprocessing (dividing by 255 to have values in [0, 1])
```

Comment:
Clarification: The processing is done _inside_ policy. E.g: "When using CNN policies, the observation is normalized inside the policy code (dividing by 255 to have values in [0, 1])".